lib/ringbuffer/ring_buffer_backend.c

```diff
@@ -12,11 +12,11 @@
 #include <linux/delay.h>
 #include <linux/errno.h>
 #include <linux/slab.h>
-#include <linux/oom.h>
 #include <linux/cpu.h>
 #include <linux/mm.h>
 #include <linux/vmalloc.h>
 
+#include <wrapper/mm.h>		/* for wrapper_check_enough_free_pages() */
```

Comment:
there are 3 wrappers implemented in the header, no need to spell out one of them...